yt_dlp/extractor/ntvcojp.py

```diff
@@ -1,55 +1,83 @@
+    IE_NAME = 'ntvjp:cu'
+    IE_DESC = '日テレ無料TADA!'
+
+    _VALID_URL = r'https?://cu\.ntv\.co\.jp/(?!program-list)(?P<id>[\w-]+)/?$'
+    _TESTS = [{
+        'url': 'https://cu.ntv.co.jp/gaki_20250525/',
         'info_dict': {
-            'id': '5978891207001',
+            'id': 'gaki_20250525',
             'ext': 'mp4',
-            'title': '桜エビと炒り卵がポイント！ 「中華風 エビチリおにぎり」──『美虎』五十嵐美幸',
-            'upload_date': '20181213',
-            'description': 'md5:1985b51a9abc285df0104d982a325f2a',
-            'uploader_id': '3855502814001',
-            'timestamp': 1544669941,
-        },
-        'params': {
-            # m3u8 download
-            'skip_download': True,
+            'title': '放送開始36年!方正ココリコが選ぶ神回&地獄回!',
+            'cast': 'count:2',
+            'description': 'md5:1e1db556224d627d4d2f74370c650927',
+            'display_id': 'ref:gaki_20250525',
+            'duration': 1450,
+            'episode': '放送開始36年!方正ココリコが選ぶ神回&地獄回!',
+            'episode_id': '000000010172808',
+            'episode_number': 255,
+            'genres': ['variety'],
+            'live_status': 'not_live',
+            'modified_date': '20250525',
+            'modified_timestamp': 1748145537,
+            'release_date': '20250525',
+            'release_timestamp': 1748145539,
+            'series': 'ダウンタウンのガキの使いやあらへんで！',
+            'series_id': 'gaki',
+            'thumbnail': r're:https?://.+\.jpg',
+            'timestamp': 1748145197,
+            'upload_date': '20250525',
+            'uploader': '日本テレビ放送網',
+            'uploader_id': '0x7FE2',
         },
-    }
-
-    BRIGHTCOVE_URL_TEMPLATE = 'http://players.brightcove.net/%s/default_default/index.html?videoId=%s'
+    }]
 
     def _real_extract(self, url):
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
-        player_config = self._search_nuxt_data(webpage, display_id)
-        video_id = traverse_obj(player_config, ('movie', 'video_id'))
-        if not video_id:
-            raise ExtractorError('Failed to extract video ID for Brightcove')
-        account_id = traverse_obj(player_config, ('player', 'account')) or '3855502814001'
-        title = traverse_obj(player_config, ('movie', 'name'))
-        if not title:
-            og_title = self._og_search_title(webpage, fatal=False) or traverse_obj(player_config, ('player', 'title'))
-            if og_title:
-                title = og_title.split('(', 1)[0].strip()
-        description = (traverse_obj(player_config, ('movie', 'description'))
-                       or self._html_search_meta(['description', 'og:description'], webpage))
+
+        info = traverse_obj(self._search_json(
+            r'window\.app\s*=', webpage, 'video info', display_id,
+        ), ('falcorCache', 'catalog', 'episode', display_id, 'value', {dict}), default={})
+        media_id = traverse_obj(info, (
+            'streaks_data', 'mediaid', {str_or_none}, {require('mediaID for Streaks')}))
```

Comment:
IMO don't bother with `traverse_obj` if the rest of the extractor is broken without the data you are traversing for (e.g. `info`)

Use `traverse_obj` when you *need* to *safely* traverse an object, or when you need its filtering/transformations/versatility

```suggestion
        info = self._search_json(
            r'window\.app\s*=', webpage, 'video info',
            display_id)['falcorCache']['catalog']['episode'][display_id]['value']
        media_id = traverse_obj(info, (
            'streaks_data', 'mediaid', {str_or_none}, {require('Streaks media ID')}))
```